src/app.js

```diff
@@ -5,8 +5,7 @@
 
 const { createServer } = require('./createServer');
 
-createServer()
-  .listen(5701, () => {
-    console.log(`Server is running on http://localhost:${5701} 🚀`);
-    console.log('Available at http://localhost:5700');
-  });
+createServer().listen(5701, () => {
+  console.log(`Server is running on http://localhost:${5701} 🚀`);
```

Comment:
If u have to repeat yourself. Its better to store value in variable. u can do it like
const PORT = process.env.PORT || 5701
and then u will be ready for production too :)